Mathlib/Topology/ContinuousFunction/Algebra.lean

```diff
@@ -220,20 +225,21 @@ theorem div_comp [Div γ] [ContinuousDiv γ] (f g : C(β, γ)) (h : C(α, β)) :
 
 -- ### "zpow" and "zsmul"
 instance hasZsmul [AddGroup β] [TopologicalAddGroup β] : SMul ℤ C(α, β)
-    where smul z f := ⟨z • f, f.Continuous.zsmul z⟩
+    where smul z f := ⟨SMul.smul z  f, f.continuous.zsmul z⟩
```

Comment:
You should never write `SMul.smul`, as that's not what `•` is syntax for in Lean 4.

Reply:
@eric-wieser thanks - still struggling with [this one](https://github.com/leanprover-community/mathlib4/pull/3332/files#diff-fea023be112f176cf1013bffd82df64e4459f2f386651e5f827e76184e48e0c1R843) though:

```
    use v x • (1 : C(α, 𝕜)) #
application type mismatch
  Exists.intro (v x • 1)
argument
  v x • 1
has type
  C(α, 𝕜) : outParam (Type (max u_1 u_2))
but is expected to have type
  ↑↑s : Type (max u_1 u_2)Lean 4
```

Reply:
I have a mathlib3 PR in the works that cleans this proof up, which might be the easy way out here

Reply:
#18888